udemy_exercises/songs/Album.java

```diff
@@ -1,19 +1,52 @@
-package udemy_exercises.songs;
+import java.util.ArrayList;
+import java.util.LinkedList;
 
-class Album {
+public class Album {
+    private String name;
+    private String artist;
+    private ArrayList<Song> songs;
 
-}
+    public Album(String name, String artist) {
+        this.name = name;
+        this.artist = artist;
+        this.songs = new ArrayList<Song>();
+    }
+
+    public boolean addSong(String title, double duration) {
+        if (findSong(title) == null) {
+            this.songs.add(new Song(title, duration));
+            return true;
+        }
+        return false;
+    }
+
+    private Song findSong(String title) {
+        for (Song checkedSong : this.songs) {
+            if (checkedSong.getTitle().equals(title)) {
```

Comment:
Consider what may happen here if a a Song were to be created with a 'title' value of [null]?  How might this line be refactored to avoid this issue?